docs/userguide/containers/dockervolumes.md

```diff
@@ -303,18 +323,38 @@ we'll be left with a backup of our `dbdata` volume.
 You could then restore it to the same container, or another that you've made
 elsewhere. Create a new container.
 
-    $ docker run -v /dbdata --name dbstore2 ubuntu /bin/bash
+```
+$ docker run -v /dbdata --name dbstore2 ubuntu /bin/bash
+```
 
 Then un-tar the backup file in the new container's data volume.
 
-    $ docker run --rm --volumes-from dbstore2 -v $(pwd):/backup ubuntu bash -c "cd /dbdata && tar xvf /backup/backup.tar --strip 1"
+```
+$ docker run --rm --volumes-from dbstore2 -v $(pwd):/backup ubuntu bash -c "cd /dbdata && tar xvf /backup/backup.tar --strip 1"
+```
 
 You can use the techniques above to automate backup, migration and
 restore testing using your preferred tools.
 
+## Removing Volumes
+
+A Docker data volume persists after a container is deleted. When you use the `docker
+run` command to add a volume to a container, you can prefix the `-v` flag with
+the `--rm` option, for example:
+
+```
+$ docker run --rm -v /foo -v awesome:/bar busybox top,
+```
+
+In this example, the `--rm` prefixes the `/foo` volume but not the `/bar`
```

Comment:
@cpuguy83 Thanks for the pointer.  I did a search, this is actually the first and only mention of anonymous volumes (however obliquely) in our documentation. Is there another term for this?  Also, there appears to be issues with container startup if these anonymous volumes are created via  the `VOLUME` tag (https://jpetazzo.github.io/2015/01/19/dockerfile-and-data-in-volumes/).  

Can you confirm that "anonymous volume" is the name for this thing and that there is no other? I'm going to open a doc issue for tracking. (I'm working on your feedback in light of this.)